src/main/java/com/wolox/training/models/Book.java

```diff
@@ -130,6 +133,7 @@ public String getAuthor() {
     }
 
     public void setAuthor(String author) {
+        Preconditions.checkNotNull(author, "The author can not be null");
```

Comment:
With strings you should also check that the parameter is not empty, for example. "" is not a valid name 😅